### ISSUE TITLE
Disconnect crashing the app on all Android versions - Using Latest Opentok 2.12

### ISSUE THREAD

**Issue by krokhale** [callstack/react-native-opentok#78]
Just tested. The call disconnect is crashing on all android versions in the latest react-native-opentok 2.12

Both .disconnectAll and .disconnect are crashing.

**Comment by renanpupin** [comment]
Same problem here, I downgraded the SDK to 2.10 and everything works fine but it will be nice if we can fix that in the latest version.

**Comment by krokhale** [comment]
True, but 2.10 crashes on Oreo, so it's now a choice between the two.

**Comment by wzywno** [comment]
Hi @krokhale @renanpupin 
Did you resolve this issue or find any workaround?

**Comment by HaroldoTeruya** [comment]
With the 2.13.
I was debugging here and found some tips but nothing that resolve the problem.

- In RNOpenTokPublisherView.cleanUpPublisher, when I remove/comment 'mPublisher.destroy()' line,  the application don't crash, BUT, the <Subscriber/> in the App.js freeze and eventually, I can't connect again.

**Comment by HaroldoTeruya** [comment]
@krokhale just for curiosity, what's the consequences if I remove the 'mPublisher.destroy()'?

**Comment by timhuff** [comment]
From what I can tell, `.destroy()` tries to re-free a resource that was freed when the component is removed. But it appears that it also stops the camera. When I commented it out, the camera wouldn't close. My solution was to keep it commented out but then also call `.disconnectAll`.

**Comment by timhuff** [comment]
Pull request indicating the change. https://github.com/callstack/react-native-opentok/pull/88
The readme should probably be updated to indicate the necessity of `.disconnectAll`.
